AimBot/Classes.h

```diff
@@ -1,12 +1,21 @@
 #pragma once
 
+class Point
```

Comment:
If it's not containing a method, use struct. 
https://www.fluentcpp.com/2017/06/13/the-real-difference-between-struct-class/

Reply:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-org